src/catalogue/items/catalogueItemsDialog.component.tsx

```diff
@@ -300,9 +414,30 @@ function CatalogueItemsDialog(props: CatalogueItemsDialogProps) {
     ) as CatalogueItemProperty[];
 
     const catalogueItem: AddCatalogueItem = {
+      name: catalogueItemDetails.name ?? '',
+      cost_gbp: catalogueItemDetails.cost_gbp
+        ? Number(catalogueItemDetails.cost_gbp)
+        : 0,
```

Comment:
Could this be covered in the unit tests - there are only two that aren't.

Reply:
These are impossible to trigger because of the error checking